src/components/Utils.js

```diff
@@ -125,8 +125,8 @@ export const IcseHeading = (props) => {
         innerForm={() => {
           return props.type === "subHeading" ? (
             <h5>{props.name}</h5>
-          ) : props.type === "p" ? (
-            <p>{props.name}</p>
+          ) : props.type === "section" ? (
+            <h6>{props.name}</h6>
```

Comment:
can we leave `props.type` as `p` even though we are using `h6`? i think this might cause some unintended things to break

Reply:
we could just add another `? :` block for for section and have both?

Reply:
keeping both ◡̈